c-sharp-syllabus-main/c-sharp-syllabus-main/csharp-basics/exercises/ClassesAndObjects/EnergyDrinks/Program.cs

```diff
@@ -8,28 +8,47 @@ namespace EnergyDrinks
 {
     class Program
     {
-        private const int NumberedSurveyed = 12467;
-        private const double PurchasedEnergyDrinks = 0.14;
-        private const double PreferCitrusDrinks = 0.64;
+        public int NumberedSurveyed;
+        public double PurchasedEnergyDrinks;
+        public double PreferCitrusDrinks;
+
+        private Program(int numberSurveyed, double purchasedEnergyDrinks, double preferCitrusDrinks)
+        {
+            NumberedSurveyed = numberSurveyed;
+            PurchasedEnergyDrinks = purchasedEnergyDrinks;
```

Comment:
Šis ir vajadzīgs?

Reply:
Jā! Tas ir survey konstruktors, uz kura vēlāk balstās metožu izvade.